admin/partials/requests.php

```diff
@@ -182,75 +188,72 @@
-									<input type="hidden" name="user_email" value="<?php echo esc_attr( $request['email'] ) ?>">
+									<input type="hidden" name="user_email" value="<?php echo esc_attr( $request['email'] ); ?>">
 									<input type="hidden" name="index" value="<?php echo esc_attr( $i ); ?>">
-									<?php submit_button( esc_html__( 'Cancel Request', 'gdpr' ), 'delete', '', false ) ?>
+									<?php submit_button( esc_html__( 'Cancel Request', 'gdpr' ), 'delete', '', false ); ?>
 								</form>
-								<form class="frm-process-user-deletion" action="<?php echo esc_url( admin_url('admin-post.php') ); ?>" method="post">
+								<form class="frm-process-user-deletion" action="<?php echo esc_url( admin_url( 'admin-post.php' ) ); ?>" method="post">
 									<?php wp_nonce_field( 'gdpr-request-delete-user', 'gdpr_delete_user' ); ?>
 									<input type="hidden" name="action" value="gdpr_delete_user">
-									<input type="hidden" name="user_email" value="<?php echo esc_attr( $request['email'] ) ?>">
+									<input type="hidden" name="user_email" value="<?php echo esc_attr( $request['email'] ); ?>">
 									<input type="hidden" name="index" value="<?php echo esc_attr( $i ); ?>">
-									<?php submit_button( esc_html__( 'Delete User', 'gdpr' ), 'primary', '', false ) ?>
+									<?php submit_button( esc_html__( 'Delete User', 'gdpr' ), 'primary', '', false ); ?>
 								</form>
 							</td>
 						</tr>
-						<?php if ( GDPR_Requests::user_has_content( $user ) ): ?>
+						<?php if ( GDPR_Requests::user_has_content( $user ) ) : ?>
 							<tr class="review" data-index="<?php echo esc_attr( $index ); ?>">
 								<td colspan="4">
 									<div class="hidden">
 										<table class="widefat">
 											<thead>
 												<tr>
 													<th><?php esc_html_e( 'Content Type', 'gdpr' ); ?></th>
-													<th class="text-center"><?php _e( 'Count', 'gdpr' ); ?></th>
-													<th class="text-center"><?php _e( 'Review', 'gdpr' ); ?></th>
-													<th class="text-center"><?php _e( 'Reassign', 'gdpr' ); ?></th>
-													<th class="text-center"><?php _e( 'Action', 'gdpr' ); ?></th>
+													<th class="text-center"><?php esc_html_e( 'Count', 'gdpr' ); ?></th>
+													<th class="text-center"><?php esc_html_e( 'Review', 'gdpr' ); ?></th>
+													<th class="text-center"><?php esc_html_e( 'Reassign', 'gdpr' ); ?></th>
+													<th class="text-center"><?php esc_html_e( 'Action', 'gdpr' ); ?></th>
 												</tr>
 											</thead>
 											<tbody>
 												<?php $post_types = get_post_types( array( 'public' => true ), 'objects' ); ?>
-												<?php foreach ( $post_types as $pt ): ?>
+												<?php foreach ( $post_types as $pt ) : ?>
 													<?php
-														$uid = get_user_by( 'email', $request['email'] );
-														if ( $uid && $uid instanceof WP_User ) {
-															$uid = $uid->ID;
-														}
+													$count = 0;
```

Comment:
Why have this `$count` here?

Reply:
To avoid the notice undefined variable.

Reply:
I don't think that will ever be the case. 
`count = count_user_posts( $uid, $pt->name );` always return an int. But I guess having the variable there is not an issue.